teuthology/packaging.py

```diff
@@ -422,6 +424,30 @@ def _get_config_value_for_remote(ctx, remote, config, key):
     return config.get(key)
 
 
+def _get_response(url, wait=False, sleep=15, tries=10):
+    with safe_while(sleep=sleep, tries=tries, _raise=False) as proceed:
+        while proceed():
+            resp = requests.get(url)
+            if resp.ok:
+                log.info('Package found...')
+                break
+
+            if not wait:
+                log.info(
+                    'Package is not found at: %s (got HTTP code %s)...',
+                    url,
+                    resp.status_code,
+                )
+                break
+
+            log.info(
+                'Package not there yet (got HTTP code %s), waiting...',
```

Comment:
Yeah, I could. The teuthology logs already print out the url before these would hit though. And example would look like:

```
Looking for package version: http://gitbuilder.ceph.com/ceph-deb-trusty-x86_64-basic/ref/hammer/version
Package not there yet (got HTTP code 404), waiting...
Package not there yet (got HTTP code 404), waiting...
Package not there yet (got HTTP code 404), waiting...
... (snip)

    raise VersionNotFoundError(url)
VersionNotFoundError: Failed to fetch package version from http://gitbuilder.ceph.com/ceph-deb-trusty-x86_64-basic/ref/hammer/version
```